pyaedt/maxwell.py

```diff
@@ -1,4 +1,4 @@
-"""This module contains these Maxwell classes: ``Maxwell``, ``Maxwell2d``, and ``Maxwell3d``."""
+"""This module contains these Maxwell classes: `Maxwell`, `Maxwell2d`, and `Maxwell3d`."""
```

Comment:
@PipKat @akaszynski @MaxJPRey Is there a difference in the rendered formatting between double tick marks and single tick marks? Is there a convention for when we should use one or the other?

Reply:
@jleonatti @MaxJPRey There is a rendering difference. Double tick marks is for a  literal string. Numpy doc says to use single tick for the names of variables, modules, functions, and classes.

Reply:
That's clear now. Thanks Kathy.

Reply:
@PipKat Thanks Kathy for the clarification.